app/controllers/comments_controller.rb

```diff
@@ -19,4 +21,12 @@ def create
   def comment_params
     params.require(:comment).permit(:text)
   end
+
+  def destroy
+    @comment = Comment.find(params[:id])
+    @post = @comment.post
+    @post.decrement!(:comments_counter)
+    @comment.destroy!
+    redirect_to user_post_path(id: @post.id), notice: 'Comment was successfully deleted!'
+  end
```

Comment:
- [ ] Great work on implementing the `destroy` method to delete the comment as `admin` :100: